src/pages/definePassengerType.tsx

```diff
@@ -145,7 +145,13 @@ export const getFieldsets = (errors: ErrorInfo[], passengerType?: string): Radio
         ],
         radioError: getErrorsByIds(['define-passenger-proof'], errors),
     };
-    fieldsets.push(ageRangeFieldset, proofRequiredFieldset);
+
+    fieldsets.push(ageRangeFieldset);
+
+    if (passengerType !== 'adult' && passengerType !== 'infant') {
```

Comment:
could we have something like if (!noProofRequired.includes('passengerType', 0)) so that we aren't hard coding the ones we allow or disallow? Would mean that the logic from the API is linked up with the page, and reads from the same hymn-sheet about which passengerTypes it should render the proof stuff for.

Reply:
I initially did that and it broke. Not sure if the import had an issue as the aim was to use the array in the api

Reply:
Module not found: Can't resolve 'child_process' in '/Users/tej/dev/infinity/tfn/repos/fdbt-site/node_modules/aws-param-store/lib'

Reply:
That what we get when we do the above which what i did before. Not sure why this is happening.